tests/sentry/integrations/slack/test_disable.py

```diff
@@ -152,8 +152,8 @@ def test_slow_integration_is_not_broken_or_disabled(self):
         now = datetime.now() - timedelta(hours=1)
         for i in reversed(range(10)):
             with freeze_time(now - timedelta(days=i)):
-                buffer.record_error()
-                buffer.record_success()
+                buffer._add("error")
+                buffer._add("success")
```

Comment:
Hmm, curious why we have to change the methods here? 

Ideally, any calls to protected methods shouldn't be done from outside the class itself.